addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -822,6 +847,18 @@ export class Rtc extends Record {
         if (camera) {
             await this.toggleVideo("camera");
         }
+        this.cleanups.push(
+            // only register the beforeunload event if there is a call as FireFox will not place
+            // the pages with beforeunload listeners in the bfcache.
+            subscribe(browser, "beforeunload", (event) => {
+                if (this.store.env.services["multi_tab"].isOnLastTab) {
+                    event.preventDefault();
+                    event.returnValue = _t(
+                        "You are about to leave an active call without another available tab to take over."
+                    );
```

Comment:
Can remove code for `returnValue`, AFAIK no evergreen browser allow it so this is basically dead code.